dts/msm8916/msm8916-samsung-r00.dts

```diff
@@ -25,6 +26,7 @@
 			i2c-address = <0x25>;
 		};
 	};
+
```

Comment:
I'd mention in the commit message what this commit really does: adds whitespace to increase readability.

Reply:
I think the commits just should be squashed.